src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -271,6 +272,7 @@ def compute_contact_forces(
         *,
         link_forces: jtp.MatrixLike | None = None,
         joint_force_references: jtp.VectorLike | None = None,
+        indices_of_enabled_collidable_points: npt.NDArray,
```

Comment:
Early comment: can we avoid to expose this argument? We can read it from `model`.

Reply:
Argument removed from all the `compute_contact_forces` functions.